### PR TITLE
Nonitalic Greek capital letters

### PR DESCRIPTION
**Pull request by ronkok**
This PR fixes my error in PR #1283 and makes Greek capital letters non-italic.

### CODE REVIEW

**Comment by ronkok** [comment]
Visual confirmation:
nonitalgreek

**Comment by ronkok** [comment]
Well, I'm looking for something that works in both math mode and text mode. `\mathrm` doesn't work in text mode.  We could use `textrm` in place of `\rm`. I don't have a strong preference between those two. 

Which would you prefer?

**Comment by edemaine** [comment]
Ah right. Then instead you should add braces around each. (Currently, text after the Greek letter will also be nonitalic!)

**Comment by ronkok** [comment]
Done. Thank you for pointing that out.

**Comment by edemaine** [comment]
My next worry is that these won't behave the same as other Greek letters do within e.g. `\mathit`, `\mathbf`, `\boldsymbol`. Would it be better to add them as symbols and modify the Greek letter list in `Parser.js`?

File: src/macros.js [hunk 1]
```diff
@@ -127,19 +127,19 @@ defineMacro("\u211D", "\\mathbb{R}");
...
+defineMacro("\u0396", "{\\rm{Z}}");
+defineMacro("\u0397", "{\\rm{H}}");
+defineMacro("\u0399", "{\\rm{I}}");
+defineMacro("\u039A", "{\\rm{K}}");
+defineMacro("\u039C", "{\\rm{M}}");
+defineMacro("\u039D", "{\\rm{N}}");
+defineMacro("\u039F", "{\\rm{O}}");
+defineMacro("\u03A1", "{\\rm{P}}");
+defineMacro("\u03A4", "{\\rm{T}}");
+defineMacro("\u03A7", "{\\rm{X}}");
```
Comment (edemaine) [inline src/macros.js]: Sorry, could you also remove the braces around the letter? We just need `{\\rm A}` for example.

**Comment by ronkok** [comment]
> Would it be better to add them as symbols

Yes, it would be better to add them as symbols. But in order for that to work, KaTeX font glyphs and font metrics are needed.  Otherwise, the code in `buildCommon.js` will not work.

Perhaps it would be better to close this PR and wait until someone adds the glyphs to the fonts. I don't personally think that these particular characters are the most urgent issue we have right now.

**Comment by edemaine** [comment]
Given that I already merged #1283 (oops), we may want to still merge this... could use a second opinion, though.

**Comment by ronkok** [comment]
> Given that I already merged #1283 

I would be happy to turn in a PR that reverts #1283. I agree that it would be good to get another opinion first.

**Pull request closed without merging**